client/js/app/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM  node:lts-alpine
+
+COPY . /app
+
+WORKDIR /app
+
+RUN yarn install && \
+  yarn build
+
+CMD [ "yarn", "dev", "--host" ]
```

Comment:
This should be `yarn preview --port 3000 --host`, dev will start server with hot module reload, it wont use the built code from `yarn build`.

Consider adding `--port 3000` here: https://github.com/vespa-engine/vespa/blob/6cb5d731084c13cebdfd7e4e413143d87c9511f0/client/js/app/package.json#L8 for symmetry with `yarn dev`, then the command here only needs to replace `dev` with `preview`

Reply:
Sounds good, I updated accordingly. 😊